src/index.html

```diff
@@ -11,7 +11,164 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      class="info"
+      action="ttps://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      name="user_info"
+    >
+      <fieldset class="field_section">
+        <legend>Personal information</legend>
+        <label class="field_line">
+        Surname:
+          <input type="text"
+            name="surname"
+            autocomplete="off"
+            maxlength="20"
+            required
+          >
+        </label>
+
+        <label class="field_line">
+        Name:
+          <input
+            type="text"
+            name="name"
+            autocomplete="off"
+            maxlength="20"
+            required
+          >
+        </label>
+
+        <label class="field_line">
+        How old are You?
+          <input
+            type="number"
+            value="12"
+            name="age"
+            min="12"
+            max="100"
+            required
+          >
+        </label>
+
+        <label class="field_line">
+        Full date of birth:
+          <input
+            type="date"
+            name="birt-date"
+          >
+        </label>
+
+        <label class="field_line">
+        I accept the term of the agreement
+          <input
+            type="checkbox"
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="field_section">
+        <legend>Registration</legend>
+        <label class="field_line">
+        E-mail:
+          <input
+            type="email"
+            name="user-email"
+            placeholder="email@example.com"
+            required
+          >
+        </label>
+
+        <label class="field_line">
+            Password:
```

Comment:
exactly two indents - no more, no less
```suggestion
          Password:
```